### PR TITLE
Fixed quickstart

### PR DESCRIPTION
**Pull request by Isqander**
Fixed imports
Fixed dekPath for readFile, readPiece
decreased bucket size
cleared the instruction

### CODE REVIEW

File: ddc/developer-guide/quickstart.md [hunk 1]
```diff
@@ -14,12 +14,7 @@ Use the [🔗 Setup](setup.md) guide to create and top-up an account.
 {% tab title="JavaScript" %}
 Latest version of SDK can be found on [releases](https://github.com/Cerebellum-Network/cere-ddc-sdk-js/releases) page.
 
-```
-@cere-ddc-sdk --save @cere-ddc-sdk/ddc-client@1.3.3
-npm install --save-dev typescript
-```
-
-_**package.json**_
+Create in any directory _**package.json**_:
```
Comment (svalasovich) [inline ddc/developer-guide/quickstart.md:17]: Just `package.json: ` should be enough

File: ddc/developer-guide/quickstart.md [hunk 2]
```diff
@@ -41,6 +36,12 @@ _**package.json**_
   "type": "module"
 }
 ```
+
+Then run:
+```
+npm install --save-dev typescript
```
Comment (svalasovich) [inline ddc/developer-guide/quickstart.md:42]: Do we need typescript? We have on JS example

Comment (Isqander) [reply ddc/developer-guide/quickstart.md:42]: Don't think so - just didn't remove it from the previous version. Now will remove

File: ddc/developer-guide/quickstart.md (hunk 1 above)
Comment (Isqander) [reply ddc/developer-guide/quickstart.md]: Done

File: ddc/developer-guide/quickstart.md (hunk 2 above)
Comment (Isqander) [reply ddc/developer-guide/quickstart.md]: Done

**Pull request merged**